src/platforms/rp2040/src/lib/networkdriver.c

```diff
@@ -409,9 +411,16 @@ static term start_ap(term ap_config, GlobalContext *global)
             return BADARG_ATOM;
         }
     }
+    uint32_t channel = 0;
+    if (!term_is_invalid_term(channel_term)) {
+        channel = term_to_int32(channel_term);
+    }
```

Comment:
why not just coalesce together this and:
```
    if (channel != 0) {
        cyw43_wifi_ap_set_channel(&cyw43_state, channel);
    }
```

Reply:
That's makes much more sense.